video_test.go

```diff
@@ -40,6 +40,7 @@ var _ = Describe("Video", func() {
 				{W: 300, H: 250, ID: "1234567893-1", Pos: AdPosAboveFold, BAttr: []int{CreativeAttributeUserInitiated, CreativeAttributeWindowsDialogOrAlert}, ExpDir: []int{ExpDirRight, ExpDirDown}},
 				{W: 728, H: 90, ID: "1234567893-2", Pos: AdPosAboveFold, BAttr: []int{CreativeAttributeUserInitiated, CreativeAttributeWindowsDialogOrAlert}},
 			},
+			Placement:     1,
```

Comment:
Sorry, nitpicking, but could you use the construction here. Nothing more to add :)